.gitlab-ci.yml

```diff
@@ -173,29 +166,22 @@ Stop_review:
 # Deploy dev branch
 Dev:
   extends:
-    - .deploy-dev
+    - .deploy-main
+  stage: deploy-preview
   environment:
     name: development
   variables:
     AGENT: ${DEV_AGENT}
     NS: ${DEV_NS}
 
-Internal:
-  extends:
-    - .deploy-dev
-  environment:
-    name: internal
-  variables:
-    AGENT: ${INTERNAL_AGENT}
-    NS: ${INTERNAL_NS}
-
 Staging:
   extends:
-    - .deploy-dev
+    - .deploy-main
+  stage: deploy-preview
   environment:
     name: staging
   rules:
-    - if: $CI_COMMIT_BRANCH == "dev"
+    - if: $CI_COMMIT_BRANCH == "main"
```

Comment:
The main in already inferred via .deploy-main template. I think this is redundant.